README.md

```diff
@@ -1,11 +1,131 @@
 # datadog-api-client-python
 
-Collection of all of the Datadog public endpoints. For the official documentation, see the [Datadog API Reference](https://docs.datadoghq.com/api/).
+This repository contains a Python API client for the [Datadog API](https://docs.datadoghq.com/api/).
+The code is generated using [openapi-generator](https://github.com/OpenAPITools/openapi-generator)
+and [apigentools](https://github.com/DataDog/apigentools).
 
-This python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project.
+Datadog API Collection
 
-Need help? Reach out to our awesome [support team](https://www.datadoghq.com/support/).
+- API version: 1.0
+
+Collection of all Datadog Public endpoints.
+
+  For more information, please visit [https://www.datadoghq.com/support/](https://www.datadoghq.com/support/)
+
+*Automatically generated by the [OpenAPI Generator](https://openapi-generator.tech)*
+
+## Requirements
+
+Building and using the API client library requires:
+
+1. Python 3.6+
+
+## Installation
+
+To install the API client library, simply execute:
+
+```shell
+pip install datadog-api-client
+```
+
+## Getting Started
+
+Please follow the [installation](#installation) instruction and execute the following Python code:
+
+```python
+import os
+from dateutil.parser import parse as dateutil_parser
+import datadog_api_client.v1
+from datadog_api_client.v1.api import aws_integration_api
+from datadog_api_client.v1.models import *
+from pprint import pprint
+
+# Defining the host is optional and defaults to https://api.datadoghq.com
+# See configuration.py for a list of all supported configuration parameters.
+configuration = datadog_api_client.v1.Configuration(
+    host = "https://api.datadoghq.com"
+)
+
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+# Configure API key authorization: apiKeyAuth
+configuration.api_key['apiKeyAuth'] = os.getenv('DD_CLIENT_API_KEY')
+
+# Configure API key authorization: appKeyAuth
+configuration.api_key['appKeyAuth'] = os.getenv('DD_CLIENT_APP_KEY')
```

Comment:
Should this be `configuration.application_key`?

Reply:
Nope, it's still a `api_key` auth method